File: sdk/javascript/packages/azure/src/AzureSessionConfig.ts

export class AzureSessionConfig {
    tenantId: string;
    clientId: string;
    clientSecret: string;

    constructor(tenantId: string, clientId: string, clientSecret: string) {
        this.tenantId = tenantId;
        this.clientId = clientId;
        this.clientSecret = clientSecret;
    }
}

File: sdk/dotNet/Integrations/AzureKeyVault/AzureSessionConfig.cs
namespace AzureKeyVault
{
    public class AzureSessionConfig
    {
        public string TenantId { get; }
        public string ClientId { get; }
        public string ClientSecret { get; }

        public AzureSessionConfig(string tenantId, string clientId, string clientSecret)
        {
            TenantId = tenantId;
            ClientId = clientId;
            ClientSecret = clientSecret;
        }
    }
}